webofneeds/won-cryptography/src/main/java/won/cryptography/rdfsign/WonVerifier.java

```diff
@@ -15,6 +17,7 @@
 import won.protocol.util.WonRdfUtils;
 import won.protocol.vocabulary.WONMSG;
 
+import java.io.StringWriter;
```

Comment:
The warning is there as a reminder. When a message references the signature of another message (which is not copied into the dataset), we would have to load that message's data in order to be able to verify the signature. This step is currently skipped.